src/sentry/models/auditlogentry.py

```diff
@@ -195,6 +195,7 @@ class AuditLogEntry(Model):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_auditlogentry"
+        index_together = ("organization", "datetime")
```

Comment:
Actually, this should be a tuple of tuples

Reply:
haha black is removing them... i suppose i need a trailing comma

Reply:
Actually even better: https://docs.djangoproject.com/en/1.11/ref/models/options/#django.db.models.Options.indexes

I always forget about this because it's new in Django 1.11

Reply:
Yeah, without a trailing comma it's not actually a tuple when there's only one item 😬